configs/stage/roles/config-manager.json

```diff
@@ -39,16 +39,19 @@
       ]
     },
     {
-      "name": "RHC Viewer",
-      "display_name": "RHC viewer",
-      "description": "Can view the current configurations on RHC manager",
+      "name": "RHC User",
```

Comment:
If you are trying to change the name from `RHC Viewer` to `RHC User` can you change this PR to just update the `display_name` to the name that you want. Can you also update the version number: 

> Update roles

> When you update the role, please change the version number of the role for the service to pick up new features. If you need to update the name of a role, add or update the display_name field in the role config, as this is the the database value displayed in the UI. If you do not have this set, RBAC defaults display_name to the value of the name field in your role config.

Reply:
I'm not sure I follow. We need to change the name *and* the associated permissions.

Also, for consistency's sake, wouldn't I change both `name` and `display_name` ?

Reply:
Once a role has been seeded/exists in RBAC's database, if you update the `name` field and the `display_name` field above it is going to create a new role in our database. If you instead just update the `display_name` - it will update the name that is shown to Users without creating a new role for RBAC. It is fine to also add the permissions. I'm just asking to not update the `name` field so that a new role is not created.

Reply:
Ahh...I see. Thanks for explaining that.

It's not my favorite thing in the world, but that's how the system is built :grimacing: